<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="loadLastProjectCheck">
         <property name="text">
          <string>&amp;Load last project on start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoSaveBeforeCompileCheck">
         <property name="text">
          <string>Automatically sa&amp;ve all files before compiling</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="generalTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editorTab">
      <attribute name="title">
       <string>&amp;Code Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="editorTabLayout">
       <item>
        <widget class="QCheckBox" name="autoIndentCheck">
         <property name="text">
          <string>Automatically &amp;indent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tabIndentsCheck">
         <property name="text">
          <string>&amp;Tab indents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="backspaceUnindentsCheck">
         <property name="text">
          <string>&amp;Backspace unindents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="editorTabSeparator1">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useTabsCheck">
         <property name="text">
          <string>&amp;Use tabs</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tabsLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QWidget" name="tabsSpacer" native="true">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tabWidthLabel">
           <property name="text">
            <string>Tab &amp;width:</string>
           </property>
           <property name="buddy">
            <cstring>tabWidthSpin</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tabWidthSpin">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="tabsRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="editorTabSeparator2">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showWhitespaceCheck">
         <property name="text">
          <string>&amp;Show whitespace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showEolCheck">
         <property name="text">
          <string>Show end of &amp;line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lineNumbersCheck">
         <property name="text">
          <string>Show line &amp;numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="editorTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="emulatorTab">
      <attribute name="title">
       <string>&amp;Emulator</string>
      </attribute>
      <layout class="QVBoxLayout" name="emulatorTabLayout">
       <item>
        <widget class="QCheckBox" name="enableSoundCheck">
         <property name="text">
          <string>Enable &amp;sound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fastLoadCheck">
         <property name="text">
          <string>&amp;Fast tape loading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tapeSoundCheck">
         <property name="text">
          <string>&amp;Tape sound</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="machineLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QWidget" name="machineSpacer" native="true">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="machineLabel">
           <property name="text">
            <string>&amp;Machine:</string>
           </property>
           <property name="buddy">
            <cstring>machineCombo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="machineCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="emulatorTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="emulatorRestartNotice">
         <property name="styleSheet">
          <string notr="true">color: red</string>
         </property>
         <property name="text">
          <string>&lt;b&gt;NOTE&lt;/b&gt;: Emulator has to be restarted for changes  to take effect.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>loadLastProjectCheck</tabstop>
  <tabstop>autoSaveBeforeCompileCheck</tabstop>
  <tabstop>autoIndentCheck</tabstop>
  <tabstop>tabIndentsCheck</tabstop>
  <tabstop>backspaceUnindentsCheck</tabstop>
  <tabstop>useTabsCheck</tabstop>
  <tabstop>tabWidthSpin</tabstop>
  <tabstop>showWhitespaceCheck</tabstop>
  <tabstop>showEolCheck</tabstop>
  <tabstop>lineNumbersCheck</tabstop>
  <tabstop>enableSoundCheck</tabstop>
  <tabstop>fastLoadCheck</tabstop>
  <tabstop>tapeSoundCheck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
